databricks/koalas/series.py

```diff
@@ -166,16 +166,16 @@ def dtype(self):
 
         Examples
         --------
-        >>> s = ks.Series([1, 2, 3])  # doctest: +SKIP
-        >>> s.dtype  # doctest: +SKIP
-        dtype('int64')
+        >>> s = ks.Series([1, 2, 3])
+        >>> s.dtype
+        dtype('int64')a
```

Comment:
was this `a` type :)?

Reply:
That's on purpose to see how travis-ci works. Will revert it later.

Reply:
Travis-ci reported failure as expected. revert it.